docs/content/overview/introduction.md

```diff
@@ -133,6 +133,7 @@ Hugo boasts the following features:
 ### Additional Features
 
   * Integrated [Disqus](https://disqus.com/) comment support
+  * [txtpen](https://txtpen.com/) highlights and in-text commenting support
```

Comment:
I appreciate it. Thanks for your kind words. Maybe it'll become common in a year or so to warrant an internal template :smile: